openshift/templates/images/api/docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:18.14.0-alpine3.17
+FROM node:lts-alpine
```

Comment:
although we want it to be latest lts, we want to know exactly the version.  we are just updating to lts, but we need to know 100% what version we are using
```suggestion
FROM node:18.18.0-alpine3.17
```